### DISCUSSION TITLE
lean-asp-net-core-2-1-add-a-react-application-to-an-existing-razor-pages-application/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#56] [Blog comments]
# lean-asp-net-core-2-1-add-a-react-application-to-an-existing-razor-pages-application/

https://blogs.taiga.nl/martijn/posts/lean-asp-net-core-2-1-add-a-react-application-to-an-existing-razor-pages-application/

**Comment by disqus-comments-migrator** [comment]
<em>[OEV](https://disqus.com/by/oev123123/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juni 29 2018, 05:37</em>

---

<p>Excellent!  I'm learning React and I wanted to have some of the flexibility of Razor Pages and this tutorial showed me exactly what I needed!!</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juli 27 2018, 01:38, in reply to [OEV](https://disqus.com/by/oev123123/)'s [comment](https://github.com/martijnboland/blog/discussions/56#discussioncomment-8953050)</em>

---

<p>Glad you liked it!</p>

**Comment by disqus-comments-migrator** [comment]
<em>[Mukesh Kamath](https://disqus.com/by/mukeshkamath/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 15 2019, 02:09</em>

---

<p>In my office I have a task of displaying react components within razor pages. Although I would have liked it in Visual Studio, I don't mind this project setup in VS Code. I am going to visit this page several times in the next few hours/days until I finish the task. I hope my search in the internet for a solution to the problem at hand has ended. Thanks In Advance.</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 15 2019, 05:17, in reply to [Mukesh Kamath](https://disqus.com/by/mukeshkamath/)'s [comment](https://github.com/martijnboland/blog/discussions/56#discussioncomment-8953052)</em>

---

<p>Good luck</p>

**Reply by disqus-comments-migrator** [reply]
<em>[Mukesh Kamath](https://disqus.com/by/mukeshkamath/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 18 2019, 02:09, in reply to [Mukesh Kamath](https://disqus.com/by/mukeshkamath/)'s [comment](https://github.com/martijnboland/blog/discussions/56#discussioncomment-8953052)</em>

---

<p>I got confused between react_notes.js and react-nodes.js...between jsx and tsx also. I also got some frustration due to not following the tutorial step by step. It was too tiring to do these two projects(ASP.Net followed by react) one after the other in one sitting. After getting errors like=&gt; Module not found: Error: Can't resolve './js/react-notes.tsx' I gave up and now running github repo to atleast see what the output should look like. May be any other day I would have done it well but not the past couple of days.</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 18 2019, 10:50, in reply to [Mukesh Kamath](https://disqus.com/by/mukeshkamath/)'s [comment](https://github.com/martijnboland/blog/discussions/56#discussioncomment-8953054)</em>

---

<p>I'm sorry to hear that. Hopefully better luck next time.</p>

**Comment by disqus-comments-migrator** [comment]
<em>[Olivier Chekroun](https://disqus.com/by/olivier_chekroun/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juli 11 2019, 03:33</em>

---

<p>How do you add breakpoint capabilities in Typescript code ?</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juli 11 2019, 07:34, in reply to [Olivier Chekroun](https://disqus.com/by/olivier_chekroun/)'s [comment](https://github.com/martijnboland/blog/discussions/56#discussioncomment-8953057)</em>

---

<p>You need to enable source maps. In our case, we're doing this in webpack.config.js (config.devtool = 'eval-source-map'). Then, you can debug typescript in your browser:<br> <a href="https://uploads.disquscdn.com/images/68feef742310fd1fd2ae094122fc907ff458d816a7ccdc8eceecf73cb7a41412.png" rel="nofollow noopener" title="https://uploads.disquscdn.com/images/68feef742310fd1fd2ae094122fc907ff458d816a7ccdc8eceecf73cb7a41412.png">https://uploads.disquscdn.com/images/68feef742310fd1fd2ae094122fc907ff458d816a7ccdc8eceecf73cb7a41412.png</a></p>
